source/agent/quic/webtransport/quicTransportServer.js

```diff
@@ -0,0 +1,135 @@
+// Copyright (C) <2020> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+/*global require*/
+'use strict';
+const EventEmitter = require('events');
+const logger = require('../../logger').logger;
+const log = logger.getLogger('QuicTransportServer');
+const zeroUuid = '00000000000000000000000000000000';
+
+/* Every QUIC agent is a QuicTransportServer which accepts QuicTransport
+ * connections from clients.
+ *
+ * Events:
+ * ended - Server is ended. All Connections will be stopped.
+ * streamadded - A new stream is added, and ready to be attached to a stream
+ * pipeline. Argument: a string for publication or subscription ID.
+ */
+
+// Connection will be closed if no transport ID is received after
+// `authenticationTimeout` seconds.
+const authenticationTimeout = 10;
+
+module.exports = class QuicTransportServer extends EventEmitter {
+  constructor(addon, port) {
+    super();
+    this._server = new addon.QuicTransportServer(port);
+    this._connections = new Map(); // Key is transport ID.
+    this._streams = new Map(); // Key is content session ID.
+    this._unAuthenticatedConnections = []; // When it's authenticated, it will be moved to this.connections.
+    this._unAssociatedStreams = []; // No content session ID assgined to them.
+    this._assignedTransportIds = []; // Transport channels assigned to this server.
+    this._server.onconnection = (connection) => {
+      this._unAuthenticatedConnections.push(connection);
+      setTimeout(() => {
+        // Must be authenticated in `authenticationTimeout` seconds.
+        if (!connection.transportId) {
+          connection.close();
+        }
+      }, authenticationTimeout * 100000);
```

Comment:
*1000,000 if you would like this to be 10 seconds.

Reply:
Fixed.